sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3305,6 +3305,8 @@ object functions {
 
   /**
    * Creates a new row for each element in the given array or map column.
+   * Uses the default column name `col` for elements in the array and
+   * `key` and `value` for elements in the map unless specified otherwise.
```

Comment:
Let's also update Python doc and R doc while we're here. Take a look for functions.py and functions.R

Reply:
oh... I forgot the viewpoint...thanks.